src/ProjectTemplates/Shared/Project.cs

```diff
@@ -15,11 +15,12 @@
 using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
+using static Templates.Test.Helpers.ProcessLock;
```

Comment:
I don't think we do this elsewhere in the codebase, could we avoid it here to keep consistency with the rest of the codebase?